release/generate_changelog.py

```diff
@@ -0,0 +1,38 @@
+"""Generate changelog entries for all PRs merged since the last release."""
+import requests
+
+get = requests.get('https://api.github.com/repos/RaRe-Technologies/gensim/releases')
+get.raise_for_status()
+
+last_release_date = get.json()[0]['published_at'][:10]
+
+
+def g():
+    page = 1
+    done = False
+
+    while not done:
+        get = requests.get(
+            'https://api.github.com/repos/RaRe-Technologies/gensim/pulls',
+            params={'state': 'closed', 'page': page},
+        )
+        get.raise_for_status()
+        pulls = get.json()
+        if not pulls:
+            break
+
+        for i, pr in enumerate(pulls):
+            if pr['created_at'] < last_release_date:
```

Comment:
How does this work? PRs created before the last release do not appear in the changelog?